src/packages/@ncigdc/modern_components/ClinicalAnalysis/CohortDropdown.js

```diff
@@ -24,22 +24,34 @@ const CohortDropdown = ({
       return null;
     }
     return (
-      <DropdownItem
-        aria-label={`Switch selected set to ${name}`}
-        className="all-sets-item"
-        key={setKey}
-        onClick={() => {
-          dispatch(
-            updateClinicalAnalysisSet({
-              id: currentAnalysis.id,
-              setId: setKey,
-              setName: name,
-            })
-          );
-        }}
+      <Tooltip
+        Component={name.length > 24 ? name : null}
```

Comment:
i think we need to reduce this length check to about 20 or 21, the tooltip doesn't show on some names that have the ellipsis overflow